legend-shared-server/src/main/java/org/finos/legend/server/shared/staticserver/Server.java

```diff
@@ -113,5 +116,14 @@ public void run(org.finos.legend.server.shared.staticserver.StaticServerConfigur
             sessionHandler.setSessionCookie(staticServerConfiguration.getSessionCookie());
         }
         environment.servlets().setSessionHandler(sessionHandler);
+        makeSessionCookieSecure(environment.getApplicationContext().getServletContext());
+    }
+
+    private void makeSessionCookieSecure( ContextHandler.Context servletContext)
+    {
+        SessionCookieConfig sessionCookieConfig = servletContext.getSessionCookieConfig();
```

Comment:
Could ever be be null?

Reply:
I have seen "JSESSIONID" getting always created with Dropwizard and Jetty so ideally SessionCookieConfig should not be null.

Reply:
As discussed, added null check to avoid potential NPE.